tests/cases/standalone/common/alter/alter_table_options.sql

```diff
@@ -28,4 +28,22 @@ SHOW CREATE TABLE ato;
 
 SELECT i FROM ato;
 
+ALTER TABLE ato SET 'compaction.twcs.time_window'='2h';
+
+ALTER TABLE ato SET 'compaction.twcs.max_output_file_size'='500MB';
+
+ALTER TABLE ato SET 'compaction.twcs.max_inactive_window_files'='2';
+
+ALTER TABLE ato SET 'compaction.twcs.max_active_window_files'='2';
+
+ALTER TABLE ato SET 'compaction.twcs.max_active_window_runs'='6';
+
+ALTER TABLE ato SET 'compaction.twcs.max_inactive_window_runs'='6';
+
+SHOW CREATE TABLE ato;
+
+ALTER TABLE ato SET 'compaction.twcs.time_window'='';
+
+SHOW CREATE TABLE ato;
+
```

Comment:
verify the table after restarting
```suggestion

-- SQLNESS ARG restart=true
SHOW CREATE TABLE ato;
```

Reply:
Thanks this command caught a bug - we ensure the serialized map must contains compaction.type when other compaction options is present. To solve this we insert twice into metadata  once per twsc change once to write the compaction.type.
```
1: Unexpected, violated: Failed to open batch regions: 0: Invalid region options, missing key compaction.type in options, at src/mito2/src/region/options.rs:424:5, at /Users/lanqing/repos/greptimedb/src/datanode/src/region_server.rs:685:14
thread 'main' panicked at tests/runner/src/env.rs:400:17:
```